### ISSUE TITLE
500 Error Can't Connect to Mongoid

### ISSUE THREAD

**Issue by simonv3** [openfarmcc/OpenFarm#524]
```
An ActionView::Template::Error occurred in registrations#new:

  Could not connect to a primary node for replica set #<Moped::Cluster:70060261934320 @seeds=[<Moped::Node resolved_address="10.28.132.37:10018">]>
  app/views/devise/registrations/new.html.erb:10:in `_app_views_devise_registrations_new_html_erb__3075420709809572341_70060261713860'
```

Got lots of these overnight.

**Comment by RickCarlino** [comment]
Possible outage from the folks that provide us DB hosting is my first guess. Or, the network connection between us and them went down for a longer than usual time.

At some point, we might want to start paying for redundant / backup databases to prevent this, but I don't think there's much that can be done about it.

**Comment by simonv3** [comment]
So an hour ago I rolled back to the last version on master 739fe3e, thinking that it might have been because of some changes to reindexing that happen in 1c6bd48, and that fixed the problem. However, looking more into it, it looks like 1c6bd48 had actually deployed fine, the problem started about 8 hours ago, so a connection problem with MongoHQ is very possible. I've now deployed back to the most recent version, and everything is fine. I wonder if MongoHQ is blocking us on memory / bandwidth?

I've sent them a support e-mail.
